query_frontend/src/promql/error.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use ceresdbproto::prometheus::sub_expr::OperatorType;
+use datafusion::error::DataFusionError;
+use snafu::{Backtrace, Snafu};
+
+#[allow(clippy::large_enum_variant)]
+#[derive(Debug, Snafu)]
+#[snafu(visibility = "pub")]
```

Comment:
Necessary?

Reply:
If not pub, then other module(like remote.rs) can't use it.